cordapp-cpk2/src/main/kotlin/net/corda/plugins/cpk2/CordappUtils.kt

```diff
@@ -40,8 +40,7 @@ const val ENTERPRISE_API_GROUP = "com.r3.corda"
 
 const val CORDAPP_SEALING_SYSTEM_PROPERTY_NAME = "net.corda.cordapp.sealing.enabled"
 
-const val CPK_TASK_NAME = "cpk"
-const val CPK_FILE_EXTENSION = "cpk"
+const val CPK_FILE_EXTENSION = "jar"
 const val CPK_ARTIFACT_CLASSIFIER = "cordapp"
```

Comment:
The `CPK_ARTIFACT_CLASSIFIER` is no longer used, and should be removed.

Reply:
Removed